app.py

```diff
@@ -22,6 +22,8 @@ def index():
         configcontent += "  path: " + request.form["target_vcf"] + "\n"
         configcontent += "gp: " + "\"" + request.form["output_genotype_prob"] + "\"" + "\n"
         configcontent += "nthreads: " + request.form["num_threads"] + "\n"
+        configcontent += "REPLACEME4BINARYPLINK: " + request.form["target_binary_plink"] + "\n"
+        configcontent += "REPLACEME4PLINK: " + request.form["target_plink"] + "\n"
```

Comment:
これは、削除していただいて、def plink, def bplink などで処理するようにして
```suggestion
        configcontent += "in_bed: " + request.form["in_bed"] + "\n"
        configcontent += "out_name: " + request.form["out_name"] + "\n"
```
REPLACEME4 xxx の部分を、cwlのパラメータにしていたいだけますか。